### PR TITLE
Correct arity for BLOCKHASH

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/story/show/165760129

### CODE REVIEW

File: src/aeb_fate_generate_ops.erl [hunk 1]
```diff
@@ -104,7 +104,7 @@ ops_defs() ->
     , { 'ORIGIN',       16#5b,    1,  false,   3, [a],              origin, "Arg0 := Address of contract called by the call transaction."}
     , { 'CALLER',       16#5c,    1,  false,   3, [a],              caller, "Arg0 := The address that signed the call transaction."}
     , { 'GASPRICE',     16#5d,    1,  false,   3, [a],            gasprice, "Arg0 := The current gas price."}
-    , { 'BLOCKHASH',    16#5e,    1,  false,   3, [a],           blockhash, "Arg0 := The current blockhash."} %% TODO: Do we support has at height?
+    , { 'BLOCKHASH',    16#5e,    2,  false,   3, [a, a],        blockhash, "Arg0 := The blockhash at height."}
```
Comment (lucafavatella) [inline src/aeb_fate_generate_ops.erl]: I see that current block hash can be achieved by using this with GENERATION (for current height). Ok.

Comment (lucafavatella) [inline src/aeb_fate_generate_ops.erl]: Hash of key block starting generation? Hash of previous key block? Hash of previous block i.e. key or micro?

Comment (lucafavatella) [inline src/aeb_fate_generate_ops.erl]: Any? Or one of the most recent 256 ones?

Can height be below genesis i.e. negative? If so, what is behaviour?

Any special behavioury in state channels - on- (force progress) or off-chain?

Ref https://github.com/aeternity/protocol/blame/f1a1ca14a7f0c95aaeba765b8bec76c77f358493/contracts/aevm.md#L81-L87

Comment (gorillainduction) [reply src/aeb_fate_generate_ops.erl]: I don't think these strings are supposed to describe the full semantics of the instructions, but if they are, we need a total overhaul at some point.

I'll leave it as it is for now.

**Pull request merged**